charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -268,6 +268,9 @@ spec:
           args:
             - --csi-address=$(ADDRESS)
             - --leader-election=true
+            {{- if .Values.controller.extraCreateMetadata }}
+            - --extra-create-metadata
+            {{- end}}
```

Comment:
We have a param `extraCreateMetadata` in our [values.yaml ](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/7db7d1a745abd1473eb86e63383c119209dcfc80/charts/aws-ebs-csi-driver/values.yaml#L144) so it makes sense to add this knob. That said, we already enable it by default and I think we should consider deprecating this parameter.

Reply:
This is consistent with the existing pattern for [StorageClass tagging](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/charts/aws-ebs-csi-driver/templates/controller.yaml#L182-L184) that requires ``--extra-create-metadata`` for ``csi-provisioner``

Reply:
I'm open to deprecating extraCreateMetadata in ``values.yaml``, but that would require changing the helm chart template for ``csi-provisioner`` for StorageClass tagging, so that should be a separate PR.

Reply:
I think leaving it as-is for this PR is fine, I am also in favor of deprecating but this PR isn't the place for it.